src/lib/hooks/usePaymentTokenForDomain.ts

```diff
@@ -0,0 +1,21 @@
+//- React Imports
+import { useQuery } from 'react-query';
+
+//- Hooks Imports
+import { useZnsSdk } from './useZnsSdk';
+
+export const usePaymentTokenForDomain = (domainId: string) => {
```

Comment:
Separated ```getPaymentTokenForDomain``` and ```getPaymentTokenInfo```. However, not sure if there would be a scenario where only the payment token would be required, but that was my thinking. I can revert this change and include this in the ```usePaymentTokenInfo``` hook if needed.

Reply:
> To run I was required to comment out the following in zUI <img alt="Screenshot 2022-08-22 at 22 17 50" width="554" src="https://user-images.githubusercontent.com/39112648/186020827-67c3c1e4-9ff3-4860-8c31-8bc6c16b8315.png">

All good - this has been fixed in the npm package